config.js

```diff
@@ -74,8 +74,8 @@ module.exports = {
     secret: process.env.KEYCLOAK_SECRET
   },
   saveService: {
-    postgresDateFormat: 'YYYY-MM-DD HH:mm:ss'
-    // port: process.env.DATASERVICE_SERVICE_PORT_HTTPS,
+    postgresDateFormat: 'YYYY-MM-DD HH:mm:ss',
+    port: process.env.DATASERVICE_SERVICE_PORT_HTTPS
     // host: process.env.DATASERVICE_SERVICE_HOST &&
```

Comment:
Can you uncomment everything in this object please

Reply:
Lines 79 and 80 uncommented